hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/test/java/org/hypertrace/traceenricher/enrichment/enrichers/HttpAttributeEnricherTest.java

```diff
@@ -147,76 +146,96 @@ public void testSemicolonInQueryParam() {
   }
 
   @Test
-  public void testNoValueForQueryParam() {
+  public void testDecodeQueryParams() {
     Event e = createMockEvent();
     when(e.getHttp())
         .thenReturn(org.hypertrace.core.datamodel.eventfields.http.Http.newBuilder().setRequest(
             Request.newBuilder()
-                .setUrl("http://hypertrace.org/users?action=checkout&location=&area&&")
```

Comment:
Is this scenario (now filter cases) covered in different tests? (.filter(kv -> kv.length == 2 && !StringUtils.isEmpty(kv[0]) && !StringUtils.isEmpty(kv[1])))

Reply:
Yes, covered in `testGetQueryParamsFromUrl`